index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>portofolio: setup and mobile version skeleton</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body class="">
+    <header class="navibar"> 
+        <nav class="nav-container">
+<a class="logo" href="#">My Logo</a>
+<img class="mobile-menu" src="img/Union.png" alt=" navigation">
+
+</nav>
+    </header>
+    <section class="h-container">
+        <div class="container">
+            <h1 class="first-text">I’m Karamaga hency<br>Glade to see you!</h1> 
+        </div>
+        <p class="second-text">
+            I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience!
+             If you like what you see and have a project you need coded,
+              don’t hestiate to contact me.   
+        </p>
+        <h4 class="icon">LET’S CONNECT</h4>
+        <ul class="icon-img">
+            <li><a href="#"><img class="t-social-media" src="img/twitter.png" alt="twitter"></a></li>
+            <li><a href="#"><img class="l-social-media" src="img/linkedin_icon.png" alt="linkedIn"></a></li>
+            <li><a href="#"><img class="m-social-media" src="img/medium.png" alt="medium"></a></li>
+            <li><a href="#"><img class="g-social-media" src="img/github.png" alt="github"></a></li>
+            <li><a href="#"><img class="v-social-media" src="img/vec.png" alt="vector"></a></li>
+          </ul>
+    </section>
```

Comment:
- [ ] Please fix styling margins and widths because there are some elements overflowing from the right and try to stick as much as possible to the given Figma design.
I think you can fix that by giving all texts width percentage values.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/159484169-a00dfbc3-3605-4771-b0b5-9377321a567d.png) | ![image](https://user-images.githubusercontent.com/68796232/159484102-8b2f6c56-ce58-4816-89a1-54c835d23203.png) |